libyul/AssemblyStack.cpp

```diff
@@ -54,7 +54,7 @@ Dialect const& languageToDialect(AssemblyStack::Language _language, EVMVersion _
 	switch (_language)
 	{
 	case AssemblyStack::Language::Assembly:
-		return EVMDialect::looseAssemblyForEVM(_version);
+		return EVMDialect::strictAssemblyForEVM(_version);
```

Comment:
Why not the same case as below?

Reply:
To be honest, I misinterpreted the differences `strictAssemblyForEVM` and `...Objects`.